src/containers/Network/UpgradeStatus.tsx

```diff
@@ -192,7 +197,9 @@ export const UpgradeStatus = () => {
       })
       .catch((e) => Log.error(e))
     Promise.all([validatorsReq, nodesReq]).then(([validators, nodes]) => {
-      setAggregated(aggregateData(validators, nodes))
+      if (validators != null && nodes != null) {
```

Comment:
ditto here

Reply:
Same here, this will only update if the links work. Otherwise it will use old data